tools/packaging/guest-image/build_se_image.sh

```diff
@@ -18,9 +18,6 @@ source "$kata_root_dir/ci/lib.sh"
 source "${packaging_root_dir}/scripts/lib.sh"
 
 ARCH=${ARCH:-$(uname -m)}
-if [ $(uname -m) == "${ARCH}" ]; then
-	[ "${ARCH}" == "s390x" ] || die "Building a Secure Execution image is currently only supported on s390x."
-fi
```

Comment:
Your commit message says is fixing only typos... so maybe this is part of another commit?

Do you mind explaining why did you moved the section?

Reply:
If I don't move this section after usage function call, the output is still wrong as follows.

```
# tools/packaging/guest-image/build_se_image.sh -h
ERROR: Building a Secure Execution image is currently only supported on s390x.
```

In other words, the logic of valid check might should after usage print.

My original intention of the PR is `fix the broken usage help` as the subject shows.

Reply:
It's ok for me. Thanks a lot for your review.